rocAL/rocAL/source/pipeline/image.cpp

```diff
@@ -144,7 +144,7 @@ ImageInfo::ImageInfo(
         _height(height_),
         _color_planes(planes),
         _batch_size(batches),
-        _data_size((static_cast<uint64_t>(width_ * height_ * _batch_size * planes))),
+        _data_size((static_cast<uint64_t>(width_ * height_ * _batch_size * planes / 8) * 8 + 8)),
```

Comment:
Is the necessary ?

Reply:
This change is made to make the buffer allocation as a multiple of 8
This was discussed in older client calls and agreed upon to make this change